recipes/fpng/all/CMakeLists.txt

```
@@ -0,0 +1,30 @@
+cmake_minimum_required(VERSION 3.8)
```

```suggestion
cmake_minimum_required(VERSION 3.15)
```
3.15 is required by Conan v2 anyway.